config/config.go

```diff
@@ -145,23 +145,22 @@ type Metrics struct {
 
 func (cfg *Metrics) validateAndLog() {
 
-	metricsEnabled := false
-
 	if cfg.Type == MetricsInflux || cfg.Influx.Enabled {
 		cfg.Influx.validateAndLog()
-		metricsEnabled = true
+		cfg.Influx.Enabled = true
```

Comment:
If validation fails, does the `log.Fatalf` call in `validateAndLog` cause the program to exit?

Reply:
Yes it does. Do you want me to change to `log.Infof` and continue with disabled Influx metrics instead?

Reply:
Nope. An exit here is desired.